Fixes invalid user query.

scherztc opened the pull request:
This PR fixes the use of the Kaltura driver when the user is not a UC user.

jamesvanmil commented:
* Need to revisit global user var
* Problem with MediaEntry item when only one video
* Clear/start session? What's it doing?

scherztc commented:
@jamesvanmil :  This is ready for review again.  I addressed the concerns listed above:

Need to revisit global current_user var
Problem with MediaEntry item when only one video
Clear/start session? What's it doing?

hortongn commented on lib/browse_everything/driver/kaltura.rb:
```diff
@@ -52,6 +59,15 @@ def authorized?
+         @@entries.extend(::Kaltura::Extensions::Mash)
+         convert_array = Array.new
+         convert_array[0] = @@entries
+         @@entries = convert_array
```
Just a style thing:  

```
convert_array = Array.new
convert_array[0] = @@entries
@@entries = convert_array
```
can be accomplished with just `@@entries = [@@entries]`

scherztc replied:
Thanks.  This works!
